src/components/TimelineEvent.js

```diff
@@ -2,9 +2,18 @@ import React from 'react';
 import './TimelineEvent.css';
 import Timestamp from './Timestamp';
 
-const TimelineEvent = () => {
+const TimelineEvent = (props) => {
+  const person = props.person;
```

Comment:
To add the CSS, you'd put the <div className="event-person"> tag before line 6 and then close it at the end of the line